app/uk/gov/hmrc/eoricommoncomponent/frontend/domain/subscription/SubscriptionDetails.scala

```diff
@@ -48,7 +48,8 @@ case class SubscriptionDetails(
   nameDobDetails: Option[NameDobMatchModel] = None,
   nameDetails: Option[NameMatchModel] = None,
   idDetails: Option[IdMatchModel] = None,
-  customsId: Option[CustomsId] = None
+  customsId: Option[CustomsId] = None,
+  formData: FormData = FormData()
```

Comment:
I had to create a sub-object here as add three more root-level fields broke the max no. params for a case class (22).

Reply:
Plus, I think it is clearer to group the data that is purely "form data".  I.e. data that is just being stored in order to re-populate the forms